content/manuals/desktop/release-notes.md

```diff
@@ -23,6 +23,73 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projects/51/views/1?filterQuery=) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-09-26" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.35.0" build_path="/TBD/" >}}
+
+### New
+
+- Support for [Docker Desktop on Red Hat Enterprise Linux](/manuals/desktop/install/linux/rhel.md) is now generally available.
+- Volume Backup and Share is now generally available and can be found in the **Volumes** tab.
+- Terminal support within Docker Desktop using system shells is now generally available.
+- Enhanced support for IPv6, allowing users to configure their Docker VM for IPv4-only, dual-stack, or IPv6-only operation. See [IPv6 documentation](https://docs.docker.com/go/desktop-ipv6-only).
+- Containerd image store:
```

Comment:
For the new section, opted for simplifying it like this, WDYT @dvdksn ?

Reply:
Looks good!